rclcpp/include/rclcpp/clock.hpp

```diff
@@ -137,6 +137,48 @@ class Clock
     Duration rel_time,
     Context::SharedPtr context = contexts::get_global_default_context());
 
+  /**
+   * Check if the clock is valid.
+   *
+   * An invalid clock is either a clock that is RCL_CLOCK_UNINITIALIZED or is an RCL_ROS_TIME
```

Comment:
Good catch, I'll update the docstring.

I was considering adding an `is_uninitialized` method, but decided against it to avoid splitting the userbase over which method to use (and prioritized `is_valid()` for consistency with the `wait_for_valid()` methods, which is both consistent with the ROS 1 names, and also imo a more intuitive name for initialized time.)